schwifty/iban.py

```diff
@@ -104,7 +119,7 @@ def generate(cls, country_code, bank_code, account_code):
 
         bank_code = bank_code.rjust(bank_and_branch_code_length, "0")
         account_code = account_code.rjust(account_code_length, "0")
-        iban = country_code + "??" + bank_code + account_code
+        iban = country_code + "??" + cls._calc_it_checksum_char(country_code, bank_code, account_code) + bank_code + account_code
```

Comment:
I would suggest to have a more generalized function like `_calc_bban_checksum` and select an appropriate country specific checksum generator from there. E.g.

```
def calc_bban_checksum(country_code, bank_code, account_code):
    if country_code == "IT":
        calc_it_checksum(bank_code, account_code)
    return ""
```

I would just make this a function instead of a class-method if you don't see a specific reason to do otherwise.

Reply:
Thank you for your suggestions. I made some changes: hope they're ok.
I do not understand though if I have to run black before committing.

Reply:
`black` is supposed to be run before committing. I will automatically reformat your code in order to match the style guideline. 

There is also a pre-commit-hook (that you have to configure) that will check if your changes comply to the style. The hook will reject the commit if the changes are violating the formatting rules.

Reply:
No problem, I can do that. I was just wondering, beacuse black will reformat also the original code in a different way. I don't know if I have to use specific parameters. Will try a new commit in a few minutes.